lib/fix-me.js

```diff
@@ -12,20 +12,24 @@ function FixMe() { }
 FixMe.prototype.runEngine = function(){
   var analysisFiles = [],
       config = {
-        include_paths: ["./"],
+        includePaths: ["./"],
         strings: ["FIXME", "TODO", "HACK", "XXX", "BUG"]
       },
       self = this;
 
   if (fs.existsSync('/config.json')) {
-    var overrides = JSON.parse(fs.readFileSync('/config.json'));
+    var userConfig = JSON.parse(fs.readFileSync('/config.json'));
 
-    for (var prop in overrides) {
-        config[prop] = overrides[prop];
+    config.includePaths = userConfig['include_paths'];
```

Comment:
I think you should use object notation whenever possible (e.g. `userConfig.include_paths`). It's only when the key is a variable (as in `config[prop]`) that we need to use `[]`-notation.